app/routes/measurement.routes.js

```diff
@@ -0,0 +1,15 @@
+module.exports = (app) => {
+  const measurements = require('../controllers/measurements.controller');
+
+  app.post('/measurements', measurements.create);
+
+  app.get('/measurements', measurements.findAll);
+
+  app.get('/measurements/:noteId', measurements.findOne);
```

Comment:
What is noteId? Why it is here? I feel in my bones copy-paste without understanding.